package.json

```diff
@@ -23,11 +22,10 @@
   },
   "scripts": {
     "setup": "node ./hosting/scripts/setup.js && yarn && yarn bootstrap && yarn build && yarn dev",
-    "bootstrap": "lerna bootstrap",
-    "postbootstrap": "lerna link && ./scripts/link-dependencies.sh",
+    "bootstrap": "lerna link && ./scripts/link-dependencies.sh",
     "build": "lerna run --stream build",
     "build:dev": "lerna run --stream prebuild && tsc --build --watch --preserveWatchOutput",
-    "backend:bootstrap": "./scripts/scopeBackend.sh 'lerna bootstrap' && yarn run postbootstrap",
+    "backend:bootstrap": "./scripts/scopeBackend.sh && yarn run bootstrap",
```

Comment:
We can probably remove this. I added it while trying to get the qa core suite to succeed with _only_ the backend packages. However I ended up going with the full bootstrap as parts of the API don't work without everything.

Reply:
You mean the full command?